Transient CMAKE error with the target `oehostmr`

Current build occasionally hits an error when compiling the target `oehostmr`. The example log is as follow.
```
[310/3885] Building C object host\CMakeFiles\oehostmr.dir\windows\syscall.c.obj

FAILED: host/CMakeFiles/oehostmr.dir/windows/syscall.c.obj 

C:\PROGRA~2\MICROS~1\2017\BUILDT~1\VC\Tools\MSVC\1416~1.270\bin\Hostx64\x64\cl.exe  /nologo -DOE_API_VERSION=2 -DOE_BUILD_UNTRUSTED -DOE_REPO_BRANCH_NAME=\"refs/heads/staging\" -DOE_REPO_LAST_COMMIT=\"68744d828de6301b9978c31ad0310bc0f7dffab3\" -I..\..\3rdparty\mbedtls\mbedtls\include -Ihost -I..\..\include /DWIN32 /D_WINDOWS /wd4566 /wd4200 /MDd /Zi /Ob0 /Od /RTC1   /WX /W2 -DOEHOSTMR /showIncludes /Fohost\CMakeFiles\oehostmr.dir\windows\syscall.c.obj /Fdhost\CMakeFiles\oehostmr.dir\oehostmr.pdb /FS -c ..\..\host\windows\syscall.c

..\..\host\windows\syscall.c(36): fatal error C1083: Cannot open include file: 'syscall_u.h': No such file or directory
```

This error indicates that OE didn't set the dependency properly.

--- a/host/measure/CMakeLists.txt
+++ b/host/measure/CMakeLists.txt
@@ -100,9 +100,9 @@ if (WIN32)
   target_link_libraries(oehostmr PUBLIC crypt32)
 endif ()
 
+add_dependencies(oehostmr syscall_untrusted_edl tee_untrusted_edl)
 if (OE_SGX)
-  add_custom_target(edger8r_result DEPENDS sgx_u.h switchless_u.h switchless_u.c)
-  add_dependencies(oehostmr edger8r_result)
+  add_dependencies(oehostmr sgx_untrusted_edl switchless_untrusted_edl)
 endif ()
 
 # TODO: Replace these with `find_package` and add as dependencies to

